Traceback (most recent call last):
  File "/opt/conda/envs/engr131/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/conda/envs/engr131/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/conda/envs/engr131/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/conda/envs/engr131/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/opt/conda/envs/engr131/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/opt/conda/envs/engr131/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/conda/envs/engr131/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from ENGR131_Util_2024 import engr131_midterm

# Register the log function to be called before any cell is executed
get_ipython().events.register('pre_run_cell', engr131_midterm)

# Please provide your first name, last name, Drexel ID, and Drexel email. Make sure these are provided as strings. "STRINGS ARE TEXT ENCLOSED IN QUOTATION MARKS."

# In the assignments you will see sections of code that you need to fill in that are marked with ... (three dots). Replace the ... with your code.
first_name = ...
last_name = ...
drexel_id = ...
drexel_email = ...

------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 1[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mENGR131_Util_2024[39;00m [38;5;28;01mimport[39;00m engr131_midterm
[1;32m      3[0m [38;5;66;03m# Register the log function to be called before any cell is executed[39;00m
[1;32m      4[0m get_ipython()[38;5;241m.[39mevents[38;5;241m.[39mregister([38;5;124m'[39m[38;5;124mpre_run_cell[39m[38;5;124m'[39m, engr131_midterm)

[0;31mModuleNotFoundError[0m: No module named 'ENGR131_Util_2024'

